src/sentry/api/endpoints/rule_snooze.py

```diff
@@ -121,37 +121,46 @@ def delete(self, request: Request, project, rule_id) -> Response:
                 detail="This feature is not available for this organization.",
                 code=status.HTTP_401_UNAUTHORIZED,
             )
-
-        serializer = RuleSnoozeValidator(data=request.data)
-        if not serializer.is_valid():
-            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
-
-        data = serializer.validated_data
         rule = self.get_rule(rule_id)
-        user_id = request.user.id if data.get("target") == "me" else None
 
-        if not can_edit_alert_rule(rule, project.organization, user_id, request.user):
-            raise AuthenticationFailed(
-                detail="Requesting user cannot mute this rule.", code=status.HTTP_401_UNAUTHORIZED
-            )
+        # find if there is a mute for all that I can remove
+        shared_snooze = None
+        made_delete = False
+        kwargs = {self.rule_field: rule, "user_id": None}
```

Comment:
I dont think you need to explicitly pass none here

Reply:
@ceorourke I do or else it won't check the `user_id` in the filter at all

Reply:
Oh yeah I guess like Richard's comment I was assuming it'd be a `.get` since we only expect to find one value

Reply:
@ceorourke oh yea that was supposed to be `.get` but I still have to pass `user_id` is `None`